code/datums/tips/tips.dm

```diff
@@ -0,0 +1,341 @@
+		"As a Monkey, you can crawl through air vents by alt+left clicking them. You must drop everything you are wearing and holding to do this, however.",
+		"As a Monkey, you can still wear a few human items, such as backpacks, gas masks and hats, and still have two free hands.",
+		"As the Malfunctioning AI, you should either order your cyborgs to dismantle the robotics console or blow it up yourself in order to protect them. Make sure to hunt down all those laptops too!",
+		"As a Revolutionary, you cannot convert a head of staff or someone who has a mindshield implant. Implants can however be surgically removed. Take control of medbay to keep control of conversions!",
+		"As a Revolutionary, cargo can be your best friend or your worst nightmare. In the best case scenario you will be able to order a limitless amount of guns and armor, in the worst case scenario security will take control and order a limitless number of loyalty implants to turn your fellow revolutionaries against you.",
+		"As a Revolutionary, your main power comes from how quickly you spread. Convert people as fast as you can and overwhelm the heads of staff before security can arm up.",
+		"As a Changeling, you can absorb someone by strangling them and using the Absorb verb; this gives you the ability to choose more powers, the DNA of whoever you absorbed, and the memory of the absorbed.",
+		"As a Cultist, invest in taking over xenobio, an adamantine golem army can quickly be converted into cultists and constructs.",
+		"As a Cultist, your team starts off very weak, but if necessary can quickly convert everything they have into raw power. Make sure you have the numbers and equipment to support going loud, or the cult will fall flat on its face.",
+		"As a Cultist, the Blood Boil rune will deal massive amounts of brute damage to non-cultists and some damage to fellow cultists of Nar-Sie nearby.",
+		"As a Cultist, you can create an army of manifested goons using a combination of the Manifest rune, which creates homunculi from ghosts, and the Blood Drain rune, which drains life from anyone standing on any blood drain rune.",
+		"Setting your own objectives and working to complete them is good practice, but the best antagonists will strive to do more than the bare minimum to really leave an impression.",
+		"The more obscure and underused a game mechanic is, the less likely your victims are to be able to deal with it.",
+		"Some antagonists are supposed to be extremely strong in one on one combat, stop getting mad about it.",
+		"As a Ninja, you should learn the difference between invisibility and invulnerability.",
+		"As a Ninja, you have a pretty badass sword. Use it.",
+		"A Ninja's sword can cut through many objects. Experiment!",
+		"As a Cortical Borer, a limp feather can kill you if you're outside of a host.",
+		"As a Death Commando you have only one course of action: RIP AND TEAR.",
```

Comment:
I think death commandos are HAPT now? Even then they hardly only RIP AND TEAR?

Reply:
they are 100% rip and tear